virttest/qemu_monitor.py

```diff
@@ -1939,20 +1942,27 @@ def get_backingfile(self, device):
             pass
         return backing_file
 
-    def block_mirror(self, device, target, speed, sync, format, mode,
-                     cmd="drive-mirror", correct=True):
+    def block_mirror(self, device, target, sync, cmd="drive-mirror",
```

Comment:
@PandaWei ,Thanks for your reminder.
According to my test, both 'drive_mirror' and 'drive-mirror' passed for qmp, I think 'cmd = self.correct(cmd)' line correct the command. Anyway, I should correct it back to 'drive-mirror'.